modules/@apostrophecms/schema/ui/apos/components/AposInputCheckboxes.vue

```diff
@@ -6,7 +6,16 @@
     :display-options="displayOptions"
   >
     <template #body>
+      <AposCombo
+        v-if="field?.style === 'combo' && choices.length"
```

Comment:
I kept it simple here since we have only 2 styles.
Also, AposInputCheckbox displays one AposCheckbox for each choice while we have a single component for AposCombo.

Reply:
field will always exist, no need for `?.` here.